libs/chain-signatures/contract/src/v0_state/mod.rs

```diff
@@ -62,3 +53,19 @@ pub struct MpcContractV0 {
     pub proposed_updates: ProposedUpdates,
     pub config: Config,
 }
+
+impl From<MpcContractV0> for MpcContract {
+    fn from(value: MpcContractV0) -> Self {
+        Self {
+            protocol_state: value.protocol_state,
```

Comment:
I don't think this will work. We changed the struct `ThresholdParametersVotes` in PR #372.

previously (what is released on testnet):
```Rust
pub struct ThresholdParametersVotes {
    proposal_by_account: BTreeMap<AuthenticatedParticipantId, ThresholdParameters>,
}
```

now:
```Rust
pub struct ThresholdParametersVotes {
    proposal_by_account: BTreeMap<AuthenticatedAccountId, ThresholdParameters>,
}
```